src/Filters/atoms/DateTimeBlock.tsx

```diff
@@ -96,22 +105,30 @@ interface IProps {
   setDateCallback?: (date: Date) => void
   setTimeCallback?: (date: Date) => void
   allowAfterMidnight?: boolean,
+  checkTimeValidation?: boolean,
```

Comment:
checkTimeValidation ---> allowManualTimeChange

Please change this on all places

Reply:
Done